cavern/src/main/java/org/opencadc/cavern/CavernConfig.java

```diff
@@ -184,6 +212,20 @@ public List<String> getAllocationParents() {
         return allocationParents;
     }
 
+    /**
+     * Obtain the API keys for administrative tasks.  The Key is the application client name, and the Value is the API key token.
+     * @return  Map of allocation API keys, where the key is the application client name, and the value is the API key token.
+     */
+    public Map<String, String> getAdminAPIKeys() {
+        return this.adminAPIKeys.stream().map(apiKey -> apiKey.split(":")).collect(
+            Collectors.toMap(splitKey -> splitKey[0], splitKey -> splitKey[1]));
+    }
+
+    public long getDefaultQuotaBytes() {
+        // Convert to bytes as that's what the NodeProperty expects.  This assumes that the default quota is in GiB.
+        return (long) (defaultQuotaGB * 1024L * 1024L * 1024L);
```

Comment:
No need for a method, actually awkward/misleading since the value is optional. Just use a `public final Long defaultQuota` field (can be null) and assign the computed value directly above.